proto/blueprint.proto

```diff
@@ -564,6 +564,12 @@ message Card {
   optional CardSize card_size = 44;
 
   optional int32 boost_level = 45;
+  
+  /** 
+   * Number of images in a gallery card. This is used to display gallery 
+   * metadata on card when the media type is MEDIA_TYPE_IMAGE. 
+   */
+  optional int32 gallery_image_count = 46;
```

Comment:
Do you think if it would be good to have this field in the `Article` message rather than `Card`?  It feels like something more about the content of the gallery article.  In terms of code structure, we can extract this piece of data from the item content which does not depend on the layout.

Reply:
We usually increment the field number by one so I am wondering if we have any reasons to assign `46`?

Reply:
@waisingyiu fixed. It was me being lazy - I just copied the declaration from `Card` to `Article` without changing the field number 🤦🏽‍♂️